### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by katiewoolsey**
PR to introduce typos and show editorial process

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,4 +3,4 @@ my demo repository where I'm following the help directions
 
 Now I'm adding some stuff to this file so that I can play around with the behavior in the diff and make some suggestions and comments. 
 
-Let's make some typooooos.
+Let's make some typooooos. And some more tyupoes that we can fizx.
```
Comment (florenciaparunov) [inline README.md:6]:
```suggestion
Let's make some typos. And some more typos that we can fix.
```
I found typos

**Review by florenciaparunov — CHANGES_REQUESTED** [review]
Review finished.

File: README.md (hunk 1 above)
Comment (katiewoolsey) [reply README.md:6]: Thanks so much, I knew that looked wrong! You are a great editor.

**Comment by katiewoolsey** [comment]
Merging, no merge conflicts!

**Pull request merged**